bitnami/metallb/templates/controller/rbac.yaml

```diff
@@ -0,0 +1,68 @@
+{{- if .Values.rbac.create -}}
```

Comment:
It would be nice if you can split both resources into different files just for consistency with the rest of the charts.

Reply:
Question here.
What does then the global option do?
So what is the expected behaviour when I do `.Values.rbac.create = false`.
Or is it then unsupported to do that for the whole chart.